### PR TITLE
added brush size indicator cursor

### PR DESCRIPTION
**Pull request by dianavermilya**

### CODE REVIEW

File: Paranormal/Paranormal/Editor/EditorView.swift [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (kotoole1) [inline Paranormal/Paranormal/Editor/EditorView.swift]: Nix this

File: Paranormal/Paranormal/Editor/EditorView.swift [hunk 2]
```diff
@@ -6,6 +7,77 @@ public class EditorView : NSView {
 
     @IBOutlet var delegate : EditorViewController?
 
+    private weak var oldCursor : NSCursor!
+
+    var customCursor : NSCursor?
```
Comment (kotoole1) [inline Paranormal/Paranormal/Editor/EditorView.swift]: Can we rename these to defaultCursor and editorCursor?

File: Paranormal/Paranormal/Editor/EditorView.swift [hunk 3]
```diff
@@ -6,6 +7,77 @@ public class EditorView : NSView {
...
+
+    func createCustomCursor(){
```
Comment (kotoole1) [inline Paranormal/Paranormal/Editor/EditorView.swift]: Space

File: Paranormal/Paranormal/Editor/EditorView.swift [hunk 4]
```diff
@@ -6,6 +7,77 @@ public class EditorView : NSView {
...
+            UInt(imageSize.height),  8,  0 , colorSpace, bitmapInfo)
+
+        CGContextSetStrokeColorWithColor(context, NSColor.darkGrayColor().CGColor)
+        CGContextSetLineWidth(context, 1.0)
+        CGContextStrokeEllipseInRect(context, circleBounds)
+        let image = CGBitmapContextCreateImage(context)
+        return NSImage(CGImage: image, size: imageSize)
+    }
+
+    public override func viewDidMoveToSuperview(){
```
Comment (kotoole1) [inline Paranormal/Paranormal/Editor/EditorView.swift]: space

File: Paranormal/Paranormal/Editor/EditorView.swift [hunk 5]
```diff
@@ -6,6 +7,77 @@ public class EditorView : NSView {
...
 
+    private weak var oldCursor : NSCursor!
+
+    var customCursor : NSCursor?
+
+    func createCustomCursor(){
+        let size_ = delegate?.document?.toolSettings.size
+        let zoom_ = delegate?.zoom
+        if let size = size_ {
+            if let zoom = zoom_ {
```
Comment (kotoole1) [inline Paranormal/Paranormal/Editor/EditorView.swift]: I'd remove the temp variables and make these four lines two lines.

**Comment by kotoole1** [comment]
Remove the images.

**Comment by kotoole1** [comment]
Overall looks really clean. There's a bunch of small things but (other than the editorview getting a little big) it's good overall.

After the fixes, :shipit:

**Comment by NCEghtebas** [comment]
Closing pull request and moving it to lukemetz/Paranormal

**Pull request closed without merging**